yabgp/message/attribute/community.py

```diff
@@ -70,75 +59,20 @@ def parse(self, value):
                     data=value)
         return community
 
-    @staticmethod
```

Comment:
Hi, You should really avoid this if/elif/else and do something like:
def to_string (community_type):
   return WELL_KNOW_COMMUNITY_INT_2_STR.get(community_type,'UNKNOW')